src/sagemaker/local/local_session.py

```diff
@@ -377,7 +378,9 @@ def invoke_endpoint(
 class LocalSession(Session):
     """Placeholder docstring"""
 
-    def __init__(self, boto_session=None):
+    def __init__(self, boto_session=None, s3_endpoint_url=None):
```

Comment:
it seems a little strange to me to modify this constructor but not `Session`'s

Reply:
The idea is that you would only do this in LocalSession; if you are running 'real' Sagemaker then you would use 'real' S3.

Reply:
I see. I guess that's fine then. Please update the PR title to make this clear.